### PR TITLE
Issue 27/more rds options

### PR DESCRIPTION
**Pull request by dpfeif**

### CODE REVIEW

**Comment by dpfeif** [comment]
I realize I didn't need to update the input part of the README

File: rds.tf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 resource "aws_db_instance" "airflow" {
   count                     = var.postgres_uri != "" || var.airflow_executor == "Sequential" ? 0 : 1
   name                      = replace(title(local.rds_name), "-", "")
-  allocated_storage         = 20
-  storage_type              = "standard"
-  engine                    = "postgres"
+  allocated_storage         = var.rds_allocated_storage
+  storage_type              = var.rds_storage_type
+  engine                    = var.rds_engine
   engine_version            = "11.8"
```
Comment (nicogelders) [inline rds.tf:7]: engine_version should also be a param

Comment (dpfeif) [reply rds.tf:7]: It's part of https://github.com/datarootsio/terraform-aws-ecs-airflow/pull/24, I didn't want to steal @frederikdesmedt's thunder. Should I put everything in this one?

Comment (nicogelders) [reply rds.tf:7]: nvm freddy already did it in #24

**Pull request merged**